api/v1alpha5/virtualmachine_storage_types.go

```diff
@@ -217,6 +217,13 @@ type PersistentVolumeClaimVolumeSource struct {
 	UnitNumber *int32 `json:"unitNumber,omitempty"`
 }
 
+func (pvc *PersistentVolumeClaimVolumeSource) GetVirtualControllerSharingMode() VirtualControllerSharingMode {
```

Comment:
* Folks working in Golang should be aware that a function receiver should be by-value if it is a read-only operation. Never use by-address unless you are assigning a value back to the receiver.
* Get this helper function out of here. It is not a constant, it is business logic.